sdk/driver/driver.go

```diff
@@ -153,3 +153,12 @@ func (d *Driver) Version(req *protocol.VersionRequest) *protocol.VersionResponse
 
 	return r
 }
+
+// SupportedLanguages handles a SupportedLanguagesRequest including information from the manifest.
+func (d *Driver) SupportedLanguages(req *protocol.SupportedLanguagesRequest) *protocol.SupportedLanguagesResponse {
+	return &protocol.SupportedLanguagesResponse{
+		Drivers: []protocol.DriverDetails{
```

Comment:
Not sure if we should allow drivers to list multiple languages...

Reply:
This is something to think about. For example, are python2 and 3, supported by the Python driver, different languages? (considering that they're incompatible). Also, a typescript driver could provide typescript and javascript.

Reply:
@mcuadros noticed me that a driver could stupor different languages.